src/client/websocket/WebSocketManager.js

```diff
@@ -19,7 +19,7 @@ const BeforeReadyWhitelist = [
 ];
 
 const UNRECOVERABLE_CLOSE_CODES = Object.keys(WSCodes).slice(1).map(Number);
-const UNRESUMABLE_CLOSE_CODES = [1000, 4006, 4007];
+const UNRESUMABLE_CLOSE_CODES = [1_000, 4_006, 4_007];
```

Comment:
Why not use `discord-api-types`? And same for the other codes in this and other files.

Reply:
Seems like that’s out of the scope of this PR and I’m also not very familiar with that package

Reply:
Then it's time to get familiar with it, I don't think this is out of scope at all, considering you just cherry-picked something completely out of scope yourself

Reply:
Alright I’ll look into it in a bit, would also appreciate if you could suggest the changes and I’ll add the necessary imports above.

Reply:
Addressed in the latest commit and also removed the out of scope commit, let me know if that works